components/omega/test/ocn/EosTest.cpp

```diff
@@ -0,0 +1,494 @@
+   TestEos->computeSpecVol(TestEos->SpecVol, Tarray, Sarray, Parray);
+   // LOG_INFO("EosTest, testEosLinear: produced SpecVol from Eos");
+
+   // check on all array values
+   int numMismatches = 0;
+   Kokkos::parallel_reduce(
+       "CheckSpecVolMatrix-linear",
+       Kokkos::MDRangePolicy<Kokkos::Rank<2>>({0, 0},
+                                              {Mesh->NCellsAll, NVertLevels}),
+       KOKKOS_LAMBDA(const int i, const int j, int &localCount) {
+          if (!isApprox(TestEos->SpecVol(i, j), LinearExpValue, RTol)) {
+             localCount++;
+          }
+       },
+       numMismatches);
+
+   bool allMatch = (numMismatches == 0);
+   if (!allMatch) {
+      Err++;
+      LOG_ERROR("EosTest: SpecVol Linear isApprox FAIL, "
+                "expected {}, got {} mismatches",
+                LinearExpValue, numMismatches);
+   }
+   if (Err == 0) {
+      LOG_INFO("EosTest SpecVolCalc Linear: PASS");
+   }
+
+   Eos::erase("LinearEos");
+   return Err;
+}
+
+int testEosTeos10() {
+   int Err          = 0;
+   const auto *Mesh = HorzMesh::getDefault();
+   // create Eos to test
+   Eos::create("TeosEos", Mesh, NVertLevels);
+   Eos *TestEos       = Eos::get("TeosEos");
+   TestEos->EosChoice = EosType::TEOS10Poly75t;
+
+   // create ocean state array
+   Array2DReal Sarray = Array2DReal("Sarray", Mesh->NCellsAll, NVertLevels);
+   Array2DReal Tarray = Array2DReal("Tarray", Mesh->NCellsAll, NVertLevels);
+   Array2DReal Parray = Array2DReal("Parray", Mesh->NCellsAll, NVertLevels);
+   // Use Kokkos::deep_copy to fill the entire view with the ref value
+   deepCopy(Sarray, Sa);
+   deepCopy(Tarray, Ct);
+   deepCopy(Parray, P);
+   deepCopy(TestEos->SpecVol, 0.0);
+
+   // Key calculation
+   TestEos->computeSpecVol(TestEos->SpecVol, Tarray, Sarray, Parray);
+   // LOG_INFO("EosTest - testEosTeos: produced SpecVol from Eos");
+
+   // check on all array values
+   int numMismatches = 0;
+   // LOG_INFO("Loop extent: {}, {}", Mesh->NCellsAll, NVertLevels);
+   Kokkos::parallel_reduce(
+       "CheckSpecVolMatrix-teos",
+       Kokkos::MDRangePolicy<Kokkos::Rank<2>>({0, 0},
+                                              {Mesh->NCellsAll, NVertLevels}),
+       KOKKOS_LAMBDA(const int i, const int j, int &localCount) {
+          if (!isApprox(TestEos->SpecVol(i, j), TeosExpValueVol, RTol)) {
+             localCount++;
+          }
+       },
+       numMismatches);
+
+   bool allMatch = (numMismatches == 0);
+   if (!allMatch) {
+      Err++;
+      LOG_ERROR("EosTest: Teos SpecVol isApprox FAIL, "
+                "expected {}, got {} mismatches",
+                TeosExpValueVol, numMismatches);
+   }
+   if (Err == 0) {
+      LOG_INFO("EosTest SpecVolCalc TEOS-10: PASS");
+   }
+
+   Eos::erase("TeosEos");
+   return Err;
+}
+
+void finalizeEosTest() {
+   Eos::clear();
+   HorzMesh::clear();
+   Decomp::clear();
+   Field::clear();
+   Dimension::clear();
+   MachEnv::removeAll();
+}
+
+int gswcSpecVolCheckValue() {
```

Comment:
For Omega coding conventions, we prefer functions/methods start with a verb, so all these xxxCheckValue might be better as `checkValueXxxx`

Reply:
@katsmith133 You're right that we need to add the conventions to the developer docs. Right now they're on a Confluence page from back when we first started Omega - but we're following [LLVM Conventions](https://llvm.org/docs/CodingStandards.html)

Reply:
Fixed! And tried to fix as many of these coding conventions as I could find.